### DISCUSSION TITLE
Avoid conflict between independent tailwind css files on same website

### DISCUSSION THREAD

**Discussion by osousa** [tailwindlabs/tailwindcss#8697] [Help] [answered]
Hi there!

i'm having an issue, tried for hours to look up a solution on google, and found nothing...

I have this website where tailwindcss is being used already. Then i have a js module i need to import that uses its own tailwindcss, they're clearly interfering with each other.

tried to combine the two using a postcss precompiler , didn't work. Is this the way?

P.S.: I tried also to match the tailwind.config.js files of both, to no avail

**Comment by osousa** [comment, marked as answer]
Should've read the documentation more thoroughly:

https://tailwindcss.com/docs/configuration#prefix

**Comment by boddapQ** [comment]
Thank you!
